tests/test_keyboard_shortcuts.py

```diff
@@ -7,6 +7,53 @@
 from pages.treeherder import TreeherderPage
 
 
+def test_close_open_panels(base_url, selenium):
+    """Open Treeherder, verify shortcut: 'Esc' closes filter and job panel.
+    Open Treeherder page, open Filters panel, select random job, close all
+    panels using 'esc' button, verify if all panels are closed.
+    """
+    page = TreeherderPage(selenium, base_url).open()
+
+    page.click_on_filters_panel()
+    page.select_random_job()
+
+    assert page.filter_panel_is_open
+    assert page.info_panel_is_open
```

Comment:
Agreed. I considered adding the job_details under the info_panel as they cover a similar area, but as it's not a repeating page object I decided not to- let me know if you disagree.